build-emscripten.sh

```diff
@@ -90,12 +91,12 @@ EXPORTED_FUNCTIONS=$($EMSDK/upstream/bin/llvm-nm $LIBHEIFA --format=just-symbols
 
 echo "Running Emscripten..."
 
-BUILD_FLAGS="-lembind -o libheif.js --pre-js pre.js --post-js post.js"
+BUILD_FLAGS="-lembind -o libheif.js --pre-js pre.js --post-js post.js -sWASM=$USE_WASM"
 RELEASE_BUILD_FLAGS="-O3"
 
 if [ "$STANDALONE" = "1" ]; then
     echo "Building in standalone (non-web) build mode"
-    BUILD_FLAGS="-s STANDALONE_WASM=1 -s WASM=1 -o libheif.wasm --no-entry"
+    BUILD_FLAGS="$BUILD_FLAGS -s STANDALONE_WASM=1 -s WASM=1 -o libheif.wasm --no-entry"
```

Comment:
@farindk I specifically did not extend `BUILD_FLAGS` in standalone build because if you do, it will include embind imports in the generated WASM.

Reply:
Ok. I'll undo that. I must admit that I didn't really understand the 'standalone' option.

Reply:
@farindk [Standalone](https://github.com/emscripten-core/emscripten/wiki/WebAssembly-Standalone) builds allow you to run the generated WASM outside the browser in WebAssembly runtimes such as [Wasmer](https://wasmer.io/), [WAVM](https://github.com/WAVM/WAVM), or [wasmtime](https://github.com/CraneStation/wasmtime). Personally I'm using the standalone build in the [Wazero](https://github.com/tetratelabs/wazero) runtime to be used in the go-libheif library, this allows for a go library that is fully contained (because the WASM is embedded in the library) and doesn't need external native libraries, it also allows the library to run fully sandboxed. The only downside is that it's slower than CGO, and multithreaded support is pretty bad right now, which causes the decoding to be around 3x as slow as CGO.